src/mumble/VolumeSliderWidgetAction.cpp

```diff
@@ -0,0 +1,48 @@
+// Copyright 2005-2020 The Mumble Developers. All rights reserved.
+// Use of this source code is governed by a BSD-style license
+// that can be found in the LICENSE file at the root of the
+// Mumble source tree or at <https://www.mumble.info/LICENSE>.
+
+#include "VolumeSliderWidgetAction.h"
+
+#include <cmath>
+
+// We define a global macro called 'g'. This can lead to issues when included code uses 'g' as a type or parameter name
+// (like protobuf 3.7 does). As such, for now, we have to make this our last include.
+
+
+VolumeSliderWidgetAction::VolumeSliderWidgetAction(QObject *parent)
+	: QWidgetAction(parent) {
+
+
+
+	qsVolumeSliderWidgetAction = new QSlider(Qt::Horizontal);
+
+
+	qsVolumeSliderWidgetAction->setMinimum(-60);
+	qsVolumeSliderWidgetAction->setMaximum(30);
+	qsVolumeSliderWidgetAction->setAccessibleName(tr("Volume"));
+	qsVolumeSliderWidgetAction->setSizePolicy(QSizePolicy::MinimumExpanding, QSizePolicy::MinimumExpanding);
+
+
+	this->setDefaultWidget(qsVolumeSliderWidgetAction);
+
+}
+
+void VolumeSliderWidgetAction::setDBSliderValue(float value) {
+
+	qsVolumeSliderWidgetAction->setValue(qRound(log2(value) * 6.0));
+
+}
+
+void VolumeSliderWidgetAction::displayTooltip(int value) {
+	
+	QString tooltipText;
+	if (value < 1)
+		tooltipText = QString("%1").arg(value);
+	else
+		tooltipText = QString("+%1").arg(value);
```

Comment:
```suggestion
	if (value < 1) {
		tooltipText = QString("%1").arg(value);
	} else {
		tooltipText = QString("+%1").arg(value);
	}
```

Shouldn't the first branch display a leading minus? And shouldn't the check be about `< 0`? :eyes:

Reply:
Negative Values already have the minus so I just wanted to display a + for values >0, I don't know Why I thought using <1 instead of >0 was a better choice for this . I did it to be similar to the number in front of your username.